pkgs/development/libraries/science/math/nccl/default.nix

```diff
@@ -77,7 +79,7 @@ backendStdenv.mkDerivation (finalAttrs: {
     description = "Multi-GPU and multi-node collective communication primitives for NVIDIA GPUs";
     homepage = "https://developer.nvidia.com/nccl";
     license = licenses.bsd3;
-    platforms = [ "x86_64-linux" ];
-    maintainers = with maintainers; [ mdaiter orivej ];
+    platforms = platforms.linux;
```

Comment:
This diff adds support for eg aarch64-linux -- just double checking that that is intentional?

Reply:
Yes, it is. I need to update the PR title to include that.